support/src/main/java/io/jenkins/plugins/casc/support/jobdsl/SeedJobConfigurator.java

```diff
@@ -80,7 +80,7 @@ public Class getTarget() {
 
     @Nonnull
     @Override
-    public List<Configurator> getConfigurators(ConfigurationContext context) {
+    public List<Configurator<GeneratedItems[]>> getConfigurators(ConfigurationContext context) {
```

Comment:
looks unrelated?

Reply:
No, it is very much related. The Configurator class was changed to actually need the type defined and not done through reflection. Hence the binary incompatibility fix.